client/src/app/AppTypes.res

```diff
@@ -687,19 +731,7 @@ module SavedSettings = {
     // always use withDefault or optional because the field might be missing due
     // to old editors or new fields.
     {
-      editorSettings: {
-        runTimers: withDefault(true, field("editorSettings", field("runTimers", bool)), j),
-        showHandlerASTs: withDefault(
-          false,
-          field("editorSettings", field("showHandlerASTs", bool)),
-          j,
-        ),
-        showFluidDebugger: withDefault(
-          false,
-          field("editorSettings", field("showFluidDebugger", bool)),
-          j,
-        ),
-      },
```

Comment:
I think it wasn't intentional to in-line these decoders - I moved them to the appropriate module.